assets/pages/oneoff-contributions/components/contributionsCheckoutContainer.jsx

```diff
@@ -2,6 +2,7 @@
 
 // ----- Imports ----- //
 
+import React from 'react';
```

Comment:
Should the containers know about React?

Reply:
Discussed offline, this is to allow Babel to transpile JSX.